apps/src/p5lab/P5LabView.jsx

```diff
@@ -69,10 +69,13 @@ class P5LabView extends React.Component {
   }
 
   getChannelId() {
+    let channelId;
+
     if (dashboard && dashboard.project) {
-      return dashboard.project.getCurrentId();
+      channelId = dashboard.project.getCurrentId();
     }
-    return undefined;
+
+    return channelId;
```

Comment:
nit: do we still need this change as part of the fix?

Reply:
Sorry, I haven't seen your comments when I merged the PR. :(
This is just a small improvement while working on the fix itself. Has ho impact on fixing bug, just making code simplier